audformat/core/table.py

```diff
@@ -762,36 +762,13 @@ def __add__(self, other: 'Table') -> 'Table':
             ValueError: if values in the same position do not match
 
         """
-        if self == other:
-            return self
```

Comment:
Why are you removing this? I think it covered the case `A + A` correct?

Reply:
Not anymore, because we want `_db`, `scheme_id`, etc. in the result set to `None`.

Reply:
Because it should just return a table, and because schemes are not part of the table.
It still feels buggy to me.
Maybe we should in general rethink the objects, and make tables independent of the database. But I guess this is not easy as we had this dicussion before at some point and did not chnage it ;)

Reply:
If we make `tables` independent of the database we have to store schemes and raters with the table, but that means we cannot share them between tables anymore. I was also thinking about a solution where we add the option to temporarily store scheme and rater objects with a table until it is added to a database and then move them to the database. But that means adding a lot of complicated code so I'm more in favor of implementing the proposed `update()` method, which  is much easier to implement and maintain.

Reply:
I would not move schemes and raters into the tables, but leaf them in the database.
What we have to change is that the information which table belongs to which schemes/raters is not stored inside the table, but inside the database.

Reply:
But I guess, this is what you propose here anyway. `A + A` will be different from `A` only when `A` was assigned to a database before. And if we reassing `A + A` to a database it will again be the same as `A`, correct?

Reply:
Ok, I see your point. But that means that we have to completely break the API, cause at the moment we do:

```python
table = Table(scheme_id='scheme')
db['table'] = table
```

and with your proposal have to change e.g. to:

```python
table = Table()
db.add_table('table, scheme_id='scheme')
```

Reply:
Actually the example above is not correct, cause `scheme_id` is part of `Column` at the moment, so instead of:

table = Table()
table['column'] = Column(scheme_id='scheme')
db['table'] = table
``` 

we have to do:

table = Table()
table['column'] = Column()
db.add_table('table, scheme_id={'column': 'scheme'})

Reply:
So I propose we first give the solution proposed in #48 a chance and if we are not happy with it, we can still think about a `audformat 2.0` someday :)

Reply:
BTW, it's even more complicated, because `scheme_id` is only part of `Column`, but for `Table` we have `media_id` and `split_id`.

I think we should at least think about breaking the API here, if it simplifies a lot of other stuff and makes it easier for the user to understand. But I have to first check or try to imagine how it would then be ;)

Reply:
I would also propose to first continue with the current solution.

I will create an issue regarding the `Table` discussion, remember we have also not reached `1.0.0` yet, so there is time to change it.

Reply:
I created https://github.com/audeering/audformat/issues/50